src/proof_system/quotient_poly.rs

```diff
@@ -64,18 +66,18 @@ pub(crate) fn compute(
         })
         .collect();
 
-    Polynomial::from_coefficients_vec(domain_4n.coset_ifft(&quotient))
+    Ok(Polynomial::from_coefficients_vec(
+        domain_4n.coset_ifft(&quotient),
+    ))
 }
 
 // Ensures that the circuit is satisfied
 fn compute_circuit_satisfiability_equation(
-    domain: &EvaluationDomain,
+    domain_4n: &EvaluationDomain,
```

Comment:
same here

Reply:
Same that avobe, is there any real reason to not send directly the params we've already computed and force the program to compute them again?

i know it's cheap, it's just that doesn't make any sense to repeat the same.